tests/infra/ccf.py

```diff
@@ -305,7 +303,7 @@ def create_and_add_pending_node(self, lib_name, host, args, target_node=None):
             self.consortium.wait_for_node_to_exist_in_store(
                 primary,
                 new_node.node_id,
-                timeout=JOIN_TIMEOUT,
```

Comment:
This was the original goal of this PR. When we're testing adding an _untrusted_ node, we don't need to wait so long - we can give the node a few seconds for a single join attempt, then we check its log for errors. Ideally this would be replaced by asking the new node directly for its status (+ response to its last join request), but this should shave some time off for now.